tests/drivers/can/api/testcase.yaml

```diff
@@ -0,0 +1,4 @@
+tests:
+  peripheral.can:
+    tags: driver
```

Comment:
you could add "can" tag here too so that it is easier to run just the can stuff with "sanitycheck -t can"

Reply:
Sure.